CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+## CHANGELOG
+
+### 1.2.0 (2020-07-21)
+**SDK**
+- Upgraded build setup and some internal dependencies
+- Added support for customization of user agent information
+- Added feature to support backward & forward navigation support in a mini app
+- Added support for javascript of type module
+- Added feature to obtain geolocation data in a mini app
+- Hotfix for redirection over custom scheme and http
+
+**Sample App**
+- Updated styling of the mini app list
+- Updated App's setting screen with build information
+- Demo usage of customized user agent information
+- Add navigation into mini app
+
+### 1.1.1 (2020-06-11)
+
+**SDK**
+- *Bugfix:* `select` and `date` input elements weren't working correctly.
+- Deprecated `MiniAppDisplay#getMiniAppView()` and added `MiniAppDisplay#getMiniAppView(activityContext: Context)`. You now must provide an Activity Context when retrieving the `View` for the Mini App. This is related to the bugfix for `select` and `date` inputs - if you use the deprecated method, then these elements will not work correctly.
+
+**Sample App**
+- Display first time setup instructions on first launch of App.
+
+### 1.1.0 (2020-06-02)
+
+- Added JavaScript bridge for passing data between Mini App and Host App. Your App now must implement `MiniAppMessageBridge` and provide the implementation when calling `MiniApp#create`.
+- Deprecated `MiniApp#create(info: MiniAppInfo)`. Your App should instead use `MiniApp#create(info: MiniAppInfo, miniAppMessageBridge: MiniAppMessageBridge)`.
+- Added `getUniqueId` function to `MiniAppMessageBridge`. This function should provide a unique identifier (unique to the user and device) to Mini Apps.
+- Added support for custom scheme URL redirect. The URL `mscheme.MINI_APP_ID://miniapp/index.html` can be used from within the Mini App view to redirect to the Mini App. This matches the URL used in the iOS Mini App SDK.
```

Comment:
Does it mean: `can be used to redirect from Mini App view to the Mini App` ?

Reply:
Yes, it seems that this can be updated slightly. What it is trying to convey is that a custom scheme of the type declared can be used for redirection by the mini-app.